### PR TITLE
Update gcs-connector-settings-reference.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by SushrutHarness — DISMISSED** [review]
LGTM

File: docs/platform/connectors/cloud-providers/ref-cloud-providers/gcs-connector-settings-reference.md [hunk 1]
```diff
@@ -135,15 +135,12 @@ With this option, Harness communicates with GCP indirectly through a Harness Del
...
+Select the **Connect through Harness Delegate for OIDC** option to allow Harness Delegate to communicate directly with GCP through OIDC. This option uses OIDC authentication to access public cloud resources without secrets or credentials.
 
-Currently, the OIDC connectivity mode is compatible with [Harness Cloud build infrastructure](/docs/continuous-integration/use-ci/set-up-build-infrastructure/use-harness-cloud-build-infrastructure) only, and it is behind a feature flag. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
-
-:::
 
 :::warning
 
-Currently, the OIDC connectivity mode is not compatible with Google Cloud Functions. You can't deploy Google Cloud Functions with OIDC-enabled GCP connectors. 
+Currently, the OIDC connectivity mode is not compatible with Google Cloud Functions. You can't deploy Google Cloud Functions with OIDC-enabled GCP connectors. Feature is behind feature flag: `PL_GCP_OIDC_AUTHENTICATION`
```
Comment (SushrutHarness) [inline docs/platform/connectors/cloud-providers/ref-cloud-providers/gcs-connector-settings-reference.md:143]:
```suggestion
Currently, the OIDC connectivity mode is not compatible with Google Cloud Functions. You can't deploy Google Cloud Functions with OIDC-enabled GCP connectors. Currently, this feature is behind the feature flag `PL_GCP_OIDC_AUTHENTICATION`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```

**Review by SushrutHarness — DISMISSED** [review]
LGTM. Changed FF language to be consistent with the rest of the docs.

**Review by anshul-harness — APPROVED** [review]

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**